src/wp-includes/rest-api/endpoints/class-wp-rest-attachments-controller.php

```diff
@@ -783,18 +783,36 @@ public function prepare_item_for_response( $item, $request ) {
 					}
 
 					$size_data['source_url'] = $image_src[0];
+
+					if ( empty( $size_data['sources'] ) || ! is_array( $size_data['sources'] ) ) {
+						continue;
+					}
+
+					$directory = dirname( $size_data['source_url'] );
+					foreach ( $size_data['sources'] as $mime => &$mime_details ) {
+						$source_url                 = "{$directory}/{$mime_details['file']}";
+						$mime_details['source_url'] = $source_url;
+					}
 				}
 
 				$full_src = wp_get_attachment_image_src( $post->ID, 'full' );
 
 				if ( ! empty( $full_src ) ) {
+					$directory = dirname( $full_src[0] );
+					foreach ( $data['media_details']['sources'] as $mime => &$mime_details ) {
```

Comment:
same here.

Reply:
Please check my comments for similar questions.

Reply:
@mehulkaklotar See above.